packages/editor-ui/src/api/templates.ts

```diff
@@ -42,10 +43,15 @@ export async function getWorkflows(
 	apiEndpoint: string,
 	query: { skip: number; limit: number; categories: number[]; search: string },
 	headers?: IDataObject,
-): Promise<{ totalWorkflows: number; workflows: ITemplatesWorkflow[] }> {
+): Promise<{
+	totalWorkflows: number;
+	workflows: ITemplatesWorkflow[];
+	filters: TemplateSearchFacet[];
+	out_of: number;
```

Comment:
what's `out_of`? these params don't seem used.

Reply:
Yeah, that's a leftover